src/main/java/org/liliyamammadova/hotelapplication/exception/CustomExceptionHandler.java

```diff
@@ -0,0 +1,21 @@
+package org.liliyamammadova.hotelapplication.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestControllerAdvice
+public class CustomExceptionHandler {
+
+    @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
+    @ExceptionHandler(ReservationException.class)
+    public Map<String, String> handleInvalidArgument(ReservationException exception) {
+        Map<String, String> map =  new HashMap<>();
+        map.put("errorMessage", exception.getMessage());
+        return map;
```

Comment:
could be just return Map.Of("errorMessage", exception.getMessage())?

Reply:
fixed